psiturk/amt_services.py

```diff
@@ -56,26 +51,24 @@
 ]
 MYSQL_RESERVED_WORDS = [word.lower() for word in MYSQL_RESERVED_WORDS_CAP]
 
-class MasterRequirement(Requirement):
-    def __init__(self, sandbox=False, required_to_preview=False):
-        comparator = "Exists"
-        sandbox_qualification_type_id = "2ARFPLSP75KLA8M8DH1HTEQVJT3SY6"
-        production_qualification_type_id = "2F1QJWKUDD8XADTFD2Q0G6UTO95ALH"
-        qualification_type_id = production_qualification_type_id if not sandbox else sandbox_qualification_type_id
-        super(MasterRequirement, self).__init__(qualification_type_id=qualification_type_id, comparator=comparator, required_to_preview=required_to_preview)
+PERCENT_ASSIGNMENTS_APPROVED_QUAL_ID = '000000000000000000L0'
+NUMBER_HITS_APPROVED_QUAL_ID = '00000000000000000040'
+LOCALE_QUAL_ID = '00000000000000000071'
+MASTERS_QUAL_ID = '2F1QJWKUDD8XADTFD2Q0G6UTO95ALH'
+MASTERS_SANDBOX_QUAL_ID = '2ARFPLSP75KLA8M8DH1HTEQVJT3SY6'
+
+NOTIFICATION_VERSION = '2006-05-05'
 
-class MTurkHIT(object):
+qqclass MTurkHIT(object):
```

Comment:
@shimeez what is a `qqclass`?

Reply:
Apologies, must have accidentally tried to close vim and saved it. I've updated the branch